### PR TITLE
[P2 tests][Team 5]Add Task3: Boolean OR search Tests

### PR DESCRIPTION
**Pull request by shaheming**
### What's your team number?
**Team 5**

###  What is the functionality being tested?
Test function `searchOrQuery( )` of` InvertedIndexManager` based on `flush()` and other basic functions.

### Describe your tests briefly:
 - Test1: test if multiple keywords work or not. And we set 5 as a threshold for write counter and read counter, because I think the number will increase when we call the flush() function and we don't know the execution order of test cases, so we set them all to 5.
 - Test2: test if single key words works or not.
 - Test3: test the case that the key word does not match any file.
 - Test4: test test other words combination.
 - Test5: test empty query list. The  `NullPointerException` is excepted.

### Does each test case have comments/documentation?
Yes

### CODE REVIEW

**Comment by Haixiang6123** [comment]
Your test cases are pretty good. Seems like covering most of corner cases, such as checking null value. For adding documents to the store, I think adding multiple documents and flush them in one time would be better. Also, for cleaning up the files method, if you guys can check if file is exist by using try-catch, the codes would be more robust.

File: src/test/java/edu/uci/ics/cs221/index/inverted/Team5OrSearchTest.java [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    public void Test1() throws Exception {
+        List<String> words = new ArrayList<>();
+        words.add("cat");
+        words.add("dog");
+
+        Iterator<Document> iterator = invertedList.searchOrQuery(words);
+        int counter = 0;
+        while (iterator.hasNext()) {
+            String text = iterator.next().getText();
+            assertEquals(true, text.contains("dog") || text.contains("cat"));
```
[the remark this answers is not in the document]
Comment (chenghz4) [reply src/test/java/edu/uci/ics/cs221/index/inverted/Team5OrSearchTest.java]: Hi, thanks for the advice. But I am just little bit confused about it.  I think the search result documents are only based on what keyword you want to search. In this case, we dont know the order of the result documents.
